packages/components/data-table-manager/src/data-table-settings/density-manager/density-manager.js

```diff
@@ -123,14 +123,18 @@ const DensityManager = (props) => {
 };
 
 DensityManager.displayName = 'DensityManager';
+DensityManager.defaultProps = {
+  isCondensed: false,
+  isWrappingText: false,
+};
```

Comment:
I think it makes sense to initialize those props only here, when the density manager is rendered.